docs/usage.md

```diff
@@ -0,0 +1,55 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Usage
+
+The **Playbook Execution Wizard** widget renders playbook execution logs and related comments on the wizard for a more fine-grained control on your processes.
+
+This wizard can currently with used in tandem with following widgets:
```

Comment:
@cs-abhishek-shukla : Please rephrase the sentence. Grammatically looks incorrect

Reply:
Fixed